package.json

```diff
@@ -110,6 +111,30 @@
         "onCommand:extension.brightscript.sendRemoteText"
     ],
     "contributes": {
+        "views": {
+            "debug": [
+                {
+                    "id": "rendezvousView",
+                    "name": "Rendezvous",
+                    "when": "debugType == 'brightscript'",
+                    "group": "Rendezvous"
+                }
+            ]
+        },
+        "menus": {
+            "view/title": [
+                {
+                    "command": "extension.brightscript.toggleSortingMethod",
```

Comment:
let's call it `extension.brightscript.toggleRendezvousSortingMethod`, or `extension.brightscript.rendezvous.toggleSortingMethod

Reply:
Done